src/map/pc.cpp

```diff
@@ -9037,8 +9037,8 @@ void pc_skillup(map_session_data *sd,uint16 skill_id)
 
 			lv = sd->status.skill[idx].lv;
 			range = skill_get_range2(&sd->bl, skill_id, lv, false);
-			upgradable = (lv < skill_tree_get_max(sd->status.skill[idx].id, sd->status.class_)) ? 1 : 0;
-			clif_skillup(sd,skill_id,lv,range,upgradable);
+			bool upgradable = (lv < skill_tree_get_max(sd->status.skill[idx].id, sd->status.class_)) ? true : false;
+			clif_skillup(*sd,skill_id,lv,range,upgradable);
```

Comment:
```suggestion
			clif_skillup(*sd, skill_id, lv, range, (lv < skill_tree_get_max(sd->status.skill[idx].id, sd->status.class_)) ? true : false);
```
Since we don't reuse the `upgradable` variable, shouldn't need it to stick around.